drivers/gles3/storage/material_storage.cpp

```diff
@@ -945,7 +945,7 @@ void MaterialData::update_textures(const HashMap<StringName, Variant> &p_paramet
 					}
 				} break;
 				case ShaderLanguage::TYPE_SAMPLERCUBEARRAY: {
-					ERR_PRINT_ONCE("Type: SamplerCubeArray not supported in GL Compatibility rendering backend, please use another type.");
+					ERR_PRINT_ONCE("Type: SamplerCubeArray not supported in Compatibility renderer, please use another type.");
```

Comment:
Once again, unsure about this suggestion. Perhaps these words are trimmed to keep it brief, but I see no hint of it.
```suggestion
					ERR_PRINT_ONCE("Type: SamplerCubeArray is not supported in the Compatibility renderer, please use another type.");
```